website_form_builder/README.rst

```diff
@@ -0,0 +1,106 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl
+   :alt: License: LGPL-3
+
+====================
+Website Form Builder
+====================
+
+This module provides websites the feature of adding custom forms in any page.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Have *Administration / Settings* privileges.
+#. Go to *Settings > Activate developer mode*.
+#. Go to *Settings > Technical > Database Structure > Models*.
+#. Search for the model you want to manage website form access for.
+#. When you find it, it will have a *Website Forms* section where you can:
+
+   * Allow the model to get forms, by checking *Allowed to use in forms*.
```

Comment:
We need to provide a built-in list of allowed models without depending on the corresponding modules (crm.lead if present, hr.recruitment if present...). We can use a similar approach to the one used in website_multi_theme.

Reply:
As I just mentioned above in https://github.com/OCA/website/pull/402#issuecomment-352096802, addons that depend on `website_form` already provide those, so not needed IMO.